esx_service/utils/auth_data.py

```diff
@@ -410,7 +422,8 @@ def create_default_tenant(self):
                                            name=auth.DEFAULT_TENANT, 
                                            description="This is a default tenant", 
                                            vms=[], 
-                                           privileges=[])
+                                           privileges=[],
+                                           tenant_uuid=auth.DEFAULT_TENANT_UUID)
```

Comment:
Currently, this change is just for DEFAULT_TENANT. But my change can be easily extended to named tenant if later we plan to do similar change.